scripts/build_clustered_population_layouts.py

```diff
@@ -24,13 +24,17 @@
 
     cutout = atlite.Cutout(snakemake.input.cutout)
 
-    clustered_regions = (
+    clustered_regions = gpd.GeoSeries(
         gpd.read_file(snakemake.input.regions_onshore)
         .set_index("name")
         .buffer(0)
         .squeeze()
     )
 
+    # if 1 country, 1 node
+    if clustered_regions.size == 1:
+        clustered_regions.index = [snakemake.config["countries"][0] + "0 0"]
+
```

Comment:
```suggestion
    clustered_regions = (
        gpd.read_file(snakemake.input.regions_onshore)
        .set_index("name")
        .buffer(0)
    )

```

Reply:
hey @LucieRC, could you try out this pattern instead of your proposition? It *should* result in the same output.